tests/test_network.py

```diff
@@ -0,0 +1,33 @@
+from autowebcompat import network
+from autowebcompat import utils
+from keras import backend as K
+import numpy as np
+
+
+a=np.random.rand(1,128)
+b=np.random.rand(1,128)
+
+def calc_dist(a,b):
+    dist=np.sum(np.square(a-b))
+    dist_sqroot=np.sqrt(max(dist,K.epsilon()))
+    return dist_sqroot
+
+def test_eucledian_distance():
+    image0 = K.variable(value=a)
+    image1 = K.variable(value=b)
+    dist = network.euclidean_distance([image0,image1])
+    evaluate=K.eval(dist)
+    assert(evaluate==calc_dist(a,b))
```

Comment:
Don't use calc_dist, use hardcoded values.
Also, with floats you shouldn't check for exact equality.